unittest/mytap/tap.h

```diff
@@ -240,7 +240,7 @@ void diag(char const *fmt, ...)
 */
 
 void BAIL_OUT(char const *fmt, ...)
-  __attribute__((noreturn, format(printf,1,2)));
+  ATTRIBUTE_NORETURN ATTRIBUTE_FORMAT(printf, 1, 2);
```

Comment:
> […] on a first look, `amd64-windows` can't compile:

@bnestere Somehow it implodes on the presence of `ATTRIBUTE_NORETURN`:
https://buildbot.mariadb.org/#/builders/234/builds/35902

@vaintroub you committed a3835fad0c; do you have any ideas?

Reply:
Of course. Place ATTRIBUTE_NORETURN *before* the function name. And ATTRIBUTE_FORMAT, for the same of uniformity, too

Reply:
Oh huh 😮. Most existing attributes (not necessarily these two) were after the function signature (https://github.com/MariaDB/server/pull/3360#discussion_r1712532268).

Reply:
That’s the ticket.
https://buildbot.mariadb.org/#/builders/234/builds/35905

thanks